Makefile

```diff
@@ -53,9 +52,14 @@ push-dos-database-image: # Pushes the database image to the ECR repository
 	make docker-login
 	make docker-push NAME=data
 
-populate-database: # Deploys a kubernetes job to populate the RDS database
+populate-database: # Deploys a kubernetes job to populate the RDS database - mandatory: INSTANCE_NAME=[name];
+	export INSTANCE_NAME=$(INSTANCE_NAME) && \
```

Comment:
Do we really need that line? Any make variable should become a shell variable anyway